packages/nextjs/src/tools/codegen/extract-files.ts

```diff
@@ -36,8 +35,8 @@ export const extractFiles = (args: ExtractFilesConfig) => {
       debug.common('Skipping code extraction, not in deploy context');
       return;
     }
-    if (!validateConsent()) {
-      console.log(chalk.yellow('Skipping code extraction, consent not given'));
+    if (args.scConfig.codeGenerationOptOut) {
+      debug.common('Skipping code extraction, code generation has been opted out of');
```

Comment:
This message might be udpated as well, together with property name, please, revisit